samples/scenarios-monitoring/autoscaling-rules.bicep

```diff
@@ -0,0 +1,51 @@
+param location string = resourceGroup().location
+param appPlanName string = '${uniqueString(resourceGroup().id)}asp'
+
+resource appServicePlan 'Microsoft.Web/serverfarms@2021-03-01' = {
+  name: appPlanName
+  location: location
+  sku: {
```

Comment:
Normally we'd suggest using a variable for the SKU - what do you think about doing that here?

Also for App Service, you should only specify the `name` of the SKU and the `capacity` - the `tier`, `size`, and `family` are derived from the `name`.

Reply:
I've added the variable for the SKU now :)